.changeset/two-phones-drop.md

```diff
@@ -0,0 +1,8 @@
+---
+"create-cloudflare": minor
+---
+
+feat: Update full-stack templates to leverage newly released features from dev week.
+
+- `wrangler.toml` will now be used for for deployment and previews.
+- Smart placement will be enabled by default.
```

Comment:
nitpick: the smart placement line should probably be removed for now so this doesn't land in the changelog

Reply:
Good catch, thanks 😄